zcash_client_backend/src/data_api/error.rs

```diff
@@ -36,8 +36,11 @@ pub enum Error<DataSourceError, CommitmentTreeError, SelectionError, FeeError> {
     /// An error in transaction proposal construction
     Proposal(ProposalError),
 
-    /// The proposal was structurally valid, but spending shielded outputs of prior multi-step
-    /// transaction steps is not yet supported.
+    /// The proposal was structurally valid, but tried to do one of these unsupported things:
+    /// * spend a prior shielded output;
+    /// * pay to an output pool for which the corresponding feature is not enabled;
+    /// * pay to a TEX address if the "transparent-inputs" feature is not enabled;
+    /// or exceeded an implementation limit.
```

Comment:
I still don't understand what "exceeded an implementation limit" means.

Reply:
Answered at https://github.com/zcash/librustzcash/pull/1257#discussion_r1659148828

Reply:
In #1441 this is replaced by `ProposalError::TooLarge`.

Reply:
Now merged with `DiversifierSpaceExhausted`.